ddp-workspace/projects/ddp-prion/app.yaml

```diff
@@ -0,0 +1,25 @@
+service: prion
+runtime: python27
```

Comment:
Looks weird!  But it works, since this is all static assets.  No point in using node or java, since we're not actually running anything per se, just loading static assets.